### PR TITLE
fix: various typos

### PR DESCRIPTION
**Pull request by nschonni**
## Summary

Updated the cSpell config version to the currently supported `0.2`. Re-ran accross the repo and fixed some

### CODE REVIEW

**Comment by nschonni** [comment]
FYI, used `cspell --config cSpell.json --no-progress --words-only --unique "**" > project-words.txt` to create a big text file of all the current issues so I could pick out some of the more obvious typos

**Comment by nschonni** [comment]
Rebased out the code file changes to submit separately

**Review by IEvangelist — APPROVED** [review]
Looks good to me, we'll :shipit: - thank you 🙏

File: docs/framework/network-programming/automatic-proxy-detection.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ Automatic proxy detection is a process by which a Web proxy server is identified
 > [!NOTE]
 > Applications running as an NT Service or as part of ASP.NET use the Internet Explorer proxy server settings (if available) of the invoking user. These settings may not be available for all service applications.  
   
- Proxies are configured on a per-connectoid basis. A connectoid is an item in the network connection dialog, and can be a physical network device (a modem or Ethernet card) or a virtual interface (such as a VPN connection running over a network device). When a connectoid changes (for example, a wireless connection changes an access point, or a VPN is enabled), the proxy detection algorithm is run again.  
+ Proxies are configured on a per-connection basis. A connection is an item in the network connection dialog, and can be a physical network device (a modem or Ethernet card) or a virtual interface (such as a VPN connection running over a network device). When a connection changes (for example, a wireless connection changes an access point, or a VPN is enabled), the proxy detection algorithm is run again.
```
Comment (gewarren) [inline docs/framework/network-programming/automatic-proxy-detection.md:41]: Not sure this one should have changed?

Comment (nschonni) [reply docs/framework/network-programming/automatic-proxy-detection.md:41]: Maybe not, the term didn't appear to be used anywhere else, but maybe it is an older Windows term https://www.pcmag.com/index.php/encyclopedia/term/connectoid

**Pull request merged**